hiro/cocoa/widget/hex-edit.cpp

```diff
@@ -5,43 +5,163 @@
 -(id) initWith:(hiro::mHexEdit&)hexEditReference {
   if(self = [super initWithFrame:NSMakeRect(0, 0, 0, 0)]) {
     hexEdit = &hexEditReference;
+    if (tableView = [[NSTableView alloc] initWithFrame:[self bounds]]) {
+      [tableView setDataSource:self];
+      [tableView setHeaderView:nil];
+      
+      // remove padding and stuff to fit all 16 columns
+      [tableView setIntercellSpacing:NSMakeSize(0, 0)];
+      [tableView setStyle:NSTableViewStylePlain];
```

Comment:
We currently target macOS 10.13, where explicit NSTableView styling isn't available, so this might crash on systems below macOS 11.0. You should be able to remove this line without any impact, since the automatic styling would be used which should still resolve to the default plain style.